controllers/users.controller.js

```diff
@@ -0,0 +1,13 @@
+const {getUserData} = require("../models/users.models")
+
+
+function getUsers(req, res, next) {
+    getUserData().then((users) => {
+        res.status(200).send({users})
+    })
+    .catch((err) => {
+        console.log(err)
```

Comment:
Don't forget to call next here so that if there _is_ an error, it is passed to your error handling middleware chain.